templates/settings.yaml.erb

```diff
@@ -74,6 +74,7 @@
 
 :dynflow:
   :pool_size: <%= scope.lookupvar("foreman::dynflow_pool_size") %>
+  :redis_url: <%= scope.lookupvar("foreman::config::jobs_redis_url") %>
```

Comment:
I am seeing this error when I test this with the full workflow:

```
[ WARN 2020-01-14T14:34:33 verbose]  Undefined variable '::foreman::config::jobs_redis_url'; 
```

Reply:
In the `if` a line higher it should check `foreman::jobs_manage_service` because that guards whether the variable is defined at all. I also think the entire dynflow section could be guarded. If we don't manage the service, nothing should read its content.